src/pages/FocusPage.tsx

```diff
@@ -42,6 +43,10 @@ export const FocusPage = (props: Props) => {
           <Switzerland {...plotProps} />
         </NamedSection>
       )}
+      <NamedSection title='Model'>
+        {/*TODO Should we make height optional?*/}
+        <Chen2021FitnessWidget.ShareableComponent {...plotProps} height={-1} />
```

Comment:
The height parameter does not fit. Do you have a better suggestion, or will it be part of the planned layout restructuring anyway?